src/select/select.component.ts

```diff
@@ -116,13 +132,23 @@ import { ControlValueAccessor, NG_VALUE_ACCESSOR } from "@angular/forms";
 					size="16"
 					class="cds--select__invalid-icon cds--select__invalid-icon--warning">
 				</svg>
+				<hr *ngIf="fluid" class="cds--select__divider" />
+				<div
+					*ngIf="invalid && invalidText && fluid" role="alert" class="cds--form-requirement" aria-live="polite">
+					<ng-container *ngIf="!isTemplate(invalidText)">{{invalidText}}</ng-container>
+					<ng-template *ngIf="isTemplate(invalidText)" [ngTemplateOutlet]="invalidText"></ng-template>
+				</div>
+				<div *ngIf="!invalid && warn && fluid" class="cds--form-requirement">
+					<ng-container *ngIf="!isTemplate(warnText)">{{warnText}}</ng-container>
+					<ng-template *ngIf="isTemplate(warnText)" [ngTemplateOutlet]="warnText"></ng-template>
+				</div>
```

Comment:
I think we should simplify this and wrap it in a `ng-container` with a condition that checks for `fluid`. Thoughts?

Reply:
I agree, it will group elements related to fluid layout and reduce cognitive load when going through the code.